index.html

```diff
@@ -12,38 +12,29 @@
   <div class="landing-page">
     <nav class="nav">
       <div class="nav__logo">my logo</div>
-      <div class="nav__menu"></div>
+      <div class="nav__menu-box">
+        <div class="nav__menu nav__menu--line-1"></div>
+        <div class="nav__menu nav__menu--line-2"></div>
+        <div class="nav__menu nav__menu--line-3"></div>
+      </div>
     </nav>
```

Comment:
- [ ] Please enclose this block in a `header` tag. It is the semantic way to enclose navigation bars in a `header` tag and it is also a requirement for the project 👍